lib/ExponentialBackoff.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var ExponentialBackoff = function(minimum_backoff, exponent, starting_backoff) {
+    if (minimum_backoff === undefined) {
+        minimum_backoff = 50;
+    }
+
+    if (exponent === undefined) {
+        exponent = 2.0;
+    }
+
+    if (starting_backoff === undefined) {
+        starting_backoff = minimum_backoff;
+    }
+
+    var backoff = starting_backoff;
+
+    return {
+        getBackoff: function() {
```

Comment:
[This](https://github.com/pelias/fences-slicer/blob/master/src/stats.js) might not be a fair example (because it's mine) 

You do lose the ability to have private members, but that's ok because `javascript`.